azure-maven-plugin-lib/src/main/java/com/microsoft/azure/maven/AbstractAzureMojo.java

```diff
@@ -128,6 +128,18 @@
     @Parameter(property = "failsOnError", defaultValue = "true")
     protected boolean failsOnError;
 
+    /**
+     * Use a HTTP proxy host for the Azure Auth Client
+     */
+    @Parameter(property = "httpProxyHost", readonly = false, required = false)
+    protected String httpProxyHost;
+
+    /**
+     * Use a HTTP proxy port for the Azure Auth Client
+     */
+    @Parameter(property = "httpProxyPort", defaultValue = "80", readonly = false, required = false)
+    protected int httpProxyPort = 80;

```

Comment:
@jazzyray Can we change `protected int httpProxyPort = 80;` to `protected int httpProxyPort;` here.

Because we have already set the default value in the annotation.

Also `readonly = false, required = false` is redundent since they are `false` by default.